src/azure/CodeGeneratorRba.cs

```diff
@@ -26,6 +26,17 @@ public class CodeGeneratorRba : CodeGeneratorRb
         /// Gets the usage instructions for the code generator.
         /// </summary>
         public override string UsageInstructions => @"The ""gem 'ms_rest_azure' ~> 0.7"" is required for working with generated code.";
+        
+        /// <summary>
+        /// Name of the generated sub-folder inside output directory (private property).
+        /// </summary>
+        private string generatedFolderName = "generated";
+        
+        /// <summary>
+        /// Name of the generated sub-folder inside output directory.
+        /// </summary>
+        protected override string GeneratedFolderName { get { return generatedFolderName; } set { generatedFolderName = value; } }

```

Comment:
minor: you may want to avoid setting it as I think there is no way from command line we can say the name if i remember

Reply:
is this suggestion, to remove the setter?